Polish/kadarka/kadarka.md

```diff
@@ -5,7 +5,7 @@ dobre wino i przy okazji jest jego tyle, 
 skrzatami i dobrymi duszkami, kt�re zamieszkuj� poblisk� ��k�, tam gdzie jest uj�cie wody miejskiej.
 W pobliskiej Biedronce kupujesz Kadark�.
 
-Mo�e masz ochot� przej�� si� na poblisk� ��k�?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.